app/views/content_items/case_study.html.erb

```diff
@@ -18,26 +18,30 @@
 <%= render 'govuk_publishing_components/components/notice', @content_item.withdrawal_notice_component  %>
 
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds content-bottom-margin">
-    <div class="responsive-bottom-margin">
-      <%= render 'components/figure',
-          src: @content_item.image["url"],
-          alt: @content_item.image["alt_text"],
-          credit: @content_item.image["credit"],
-          caption: @content_item.image["caption"] if @content_item.image %>
-
-      <%= render 'govuk_publishing_components/components/govspeak', {
-        direction: page_text_direction,
-      } do %>
+  <div class="govuk-grid-column-two-thirds">
+    <div class="content-bottom-margin">
+      <div class="responsive-bottom-margin">
```

Comment:
There's a lot of margin being applied around these elements. Something to consider for future tidying perhaps is to have the margin on the components themselves. This might require more work, so non-blocking - the existing margin framework might need to be added to the components being used here, but it would make the markup cleaner and the code more readable.